lib/api-calls.js

```diff
@@ -5,7 +5,14 @@ async function getReposList(octokit, name, owner) {
     q: `"${name}" user:${owner} in:file filename:package.json`
   });
 
-  return items;
+  const deduplicatedReposList = reposList.reduce((prevItem, currentItem) => {
```

Comment:
This action can work in very large organizations and therefore the array that requires reduction can get very large, so doing `find` on the same array (but one element larger) all the time can slow down execution here. 

I think it would be much better to use `filter` instead of `reduce` and `find`. I'm also not sure where did you get `reposList` from

how about 👇🏼 

```
let processedRepo = {};
const deduplicatedReposList = items.filter(repo => {
    const id = repo.repository.id;
    // filter returns only these repos that are not present in processedRepo object, and adds them there to the list
    return !processedRepo[id] && (processedRepo[id] = true);
});

return deduplicatedReposList;
```

Reply:
this looks way better, I'll make the change 👍 .
Thanks 😃

Reply:
Could you take a look again @derberg ?